data/templates/default.latex

```diff
@@ -139,13 +139,37 @@ $endif$
 \ifPDFTeX\else
   % xetex/luatex font selection
 $if(mainfont)$
-  \setmainfont[$for(mainfontoptions)$$mainfontoptions$$sep$,$endfor$]{$mainfont$}
+  $if(mainfontfallback)$
+    \ifLuaTeX
+      \usepackage{luaotfload}
+      \directlua{luaotfload.add_fallback("mainfontfallback",{
+        $for(mainfontfallback)$"$mainfontfallback$"$sep$,$endfor$
+      })}
+    \fi
+  $endif$
+  \setmainfont[$for(mainfontoptions)$$mainfontoptions$$sep$,$endfor$$if(mainfontfallback)$,RawFeature={fallback=mainfontfallback}$endif$]{$mainfont$}
```

Comment:
Shouldn't this be `fallback=$mainfontfallback$`?  What if $mainfontfallback$ is a list?

Has this code been tested?

Reply:
`mainfontfallback` is supposed to be a list:
 * Each item is a font.
 * `luaotfload` tries each font in turn, until it finds one containing the missing character.
 * This allows setting up a *chain* of fallback fonts for greater coverage 
     (e.g. try the main font, then `StixTwoText`, then `NotoColorEmoji`, then `LastResort`)

RE testing:

* I successfully used the template in a presentation today ([slides](https://ics.uu.nl/docs/vakken/b3tc/downloads-2018/TC-03-final.pdf))
* I've used similar code several times before (via `header-includes`) for typesetting literate Agda ([documentation](https://agda.readthedocs.io/en/v2.6.4.1-rc1/tools/generating-latex.html#known-pitfalls-and-issues))
* The code closely follows the [example in the `luaotfload` docs](https://tug.ctan.org/macros/luatex/generic/luaotfload/luaotfload-latex.pdf#page=19), and the similar code on [TeX SE](https://tex.stackexchange.com/a/572220)

Reply:
RE `fallback=mainfontfallback`, the lack of `$...$` is deliberate:
* `luaotfload` requires each fallback chain to have a name
* I chose the name 'mainfontfallback' for the fallback chain set by the `mainfontfallback:` YAML option
* So the literal string `mainfontfallback` (without `$...$`) is correct here